app/views/groups/index.html.erb

```diff
@@ -7,54 +7,38 @@
 <% if @groups.any? %>
   <h1 class="group_type">Joined Groups</h1>
   <% @groups.each do |group| %>
-    <% user_is_leader = user_is_leader_of?(group) %>
-
-    <% if @groups.last == group %>
-      <div class="group no_margin_bottom">
-    <% else %>
-      <div class="group">
-    <% end %>
-    <h1 class="group_name">
-      <%= link_to group.name, group %>
-      <% if user_is_leader %>
-        (<%= t('.leader') %>)
-      <% end %>
-    </h1>
+    <div class="group <%= 'no_margin_bottom' if group == @groups.last %>">
+      <h1 class="group_name">
+        <%= link_to group.name, group %>
+        (<%= t('.leader') if user_is_leader_of? group %>)
+      </h1>
 
-    <div class="notification_wrapper small_margin_bottom">
-      <strong class="tip_notifications_button">
-        <i class="fa fa-list small_margin_right"></i>
-        <%= group.group_members.count %>
-        <% if group.group_members.count == 1 %>
-          <%= t('groups.index.member') %>
-        <% else %>
-          <%= t('groups.index.members') %>
-        <% end %>
-      </strong>
-      <%= render_group_member_partial(group.group_members) %>
-    </div>
+      <div class="notification_wrapper small_margin_bottom">
+        <strong class="tip_notifications_button">
+          <i class="fa fa-list small_margin_right"></i>
+          <%= group.group_members.count %>
+          <%= member_pluralized_for(group) %>
+        </strong>
+        <%= render_group_member_partial(group.group_members) %>
+      </div>
 
-    <%= strip_tags(group.description[0..80]) %>
-    <% if group.description.length >= 80 %>
-      <%= " [...]" %>
-    <% end %>
+      <%= strip_tags(group.description[0..80]) %>
+      <%= " [...]" if group.description.length >= 80 %>
 
-    <% if user_is_leader || (!user_is_leader || group.group_members.count > 1) || (user_is_leader && group.group_members.count == 1) %>
```

Comment:
I don't think we need this if. I can't think of a case where none of those would be true.